deploy/kubernetes/deployment.yaml

```diff
@@ -38,17 +38,11 @@ spec:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
             - "--leader-election"
-            - "--leader-election-namespace=$(MY_NAMESPACE)"
```

Comment:
should this example use lease?

Reply:
Yeah it should. Originally left that out since I didn't want anyone re-deploying this on an existing deployment but I don't think that'll be a problem